starter-code/js/index.js

```diff
@@ -1,52 +1,67 @@
-function deleteItem(e){
 
-}
 
-function getPriceByProduct(itemNode){
+// function deleteItem(e){
+//   // var button = document.getElementsByClassName('delete-button');
+//   // button.onclick = function () {
 
-}
+//   // }
+// }
 
-function updatePriceByProduct(productPrice, index){
+// function getPriceByProduct(itemNode){
 
-}
+// }
+
+// function updatePriceByProduct(productPrice, index){
+
+// }
 
 function getTotalPrice() {
+  var totalPrice = document.getElementsByClassName('unit-price')[0].innerText;
+  console.log(totalPrice);
+  var quantifyOfItems= document.document.querySelector('input[placeholder]').innerText;
```

Comment:
For access to the value of an `input` you should use `.value` instead. But careful since you always will get a `string`